gossip/src/contact_info.rs

```diff
@@ -409,6 +432,16 @@ impl ContactInfo {
     }
 }
 
+impl Default for ContactInfo {
+    fn default() -> Self {
+        Self::new(
+            Pubkey::default(),
+            0, // wallclock
+            0, // shred_version
+        )
+    }
+}
```

Comment:
equivalent to:
https://github.com/anza-xyz/agave/blob/fb3bf5534/gossip/src/legacy_contact_info.rs#L118-L136